src/Compilers/CSharp/Portable/Symbols/Source/SourceNamedTypeSymbol.cs

```diff
@@ -1053,7 +1053,7 @@ internal override void AddSynthesizedAttributes(ModuleCompilationState compilati
 
             if (this.Indexers.Any())
             {
-                string defaultMemberName = this.Indexers.First().MetadataName; // UNDONE: IndexerNameAttribute
+                string defaultMemberName = this.Indexers.First().MetadataName; // TODO: IndexerNameAttribute
                 var defaultMemberNameConstant = new TypedConstant(compilation.GetSpecialType(SpecialType.System_String), TypedConstantKind.Primitive, defaultMemberName);
```

Comment:
Overall I agree with moving UNDONE -> TODO but could we do it in a different commit?  It's making this change unnecessarily bigger.